tests/semanticdb/metac.expect

```diff
@@ -2570,19 +2471,41 @@ Schema => SemanticDB v4
 Uri => NewModifiers.scala
 Text => empty
 Language => Scala
-Symbols => 3 entries
-Occurrences => 4 entries
+Symbols => 14 entries
+Occurrences => 15 entries
 
 Symbols:
-_empty_/NewModifiers. => final object NewModifiers
-_empty_/NewModifiers.A# => type A
-_empty_/NewModifiers.foo. => val method foo
+_empty_/NewModifiers$package. => final package object _empty_ extends Object { self: _empty_.type { type OpaqueB  } => +2 decls }
+_empty_/NewModifiers$package.OpaqueB# => type OpaqueB 
```

Comment:
we should probably have a new signature kind for the alias part of an opaque type definition, e.g this is `OpaqueB /* >: Nothing <: Any */ = Int`

Reply:
Could you elaborate on it?

Reply:
an opaque type RHS is a TypeBoundsTree which has three fields - lower bound, upper bound, and alias type - we should probably record the alias in the signature as well as its bounds